sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -367,44 +368,48 @@ public synchronized void onActivityCreated(
 
   @Override
   public synchronized void onActivityStarted(final @NotNull Activity activity) {
-    // The docs on the screen rendering performance tracing
-    // (https://firebase.google.com/docs/perf-mon/screen-traces?platform=android#definition),
-    // state that the tracing starts for every Activity class when the app calls .onActivityStarted.
-    // Adding an Activity in onActivityCreated leads to Window.FEATURE_NO_TITLE not
-    // working. Moving this to onActivityStarted fixes the problem.
-    activityFramesTracker.addActivity(activity);
+    if (performanceEnabled || options.isEnableActivityLifecycleBreadcrumbs()) {
+      // The docs on the screen rendering performance tracing
+      // (https://firebase.google.com/docs/perf-mon/screen-traces?platform=android#definition),
+      // state that the tracing starts for every Activity class when the app calls
+      // .onActivityStarted.
+      // Adding an Activity in onActivityCreated leads to Window.FEATURE_NO_TITLE not
+      // working. Moving this to onActivityStarted fixes the problem.
+      activityFramesTracker.addActivity(activity);
+    }
 
     addBreadcrumb(activity, "started");
```

Comment:
I think this is how it should be:
```suggestion
    if (performanceEnabled) {
      // The docs on the screen rendering performance tracing
      // (https://firebase.google.com/docs/perf-mon/screen-traces?platform=android#definition),
      // state that the tracing starts for every Activity class when the app calls
      // .onActivityStarted.
      // Adding an Activity in onActivityCreated leads to Window.FEATURE_NO_TITLE not
      // working. Moving this to onActivityStarted fixes the problem.
      activityFramesTracker.addActivity(activity);
    }
   	if (options.isEnableActivityLifecycleBreadcrumbs()) {
      addBreadcrumb(activity, "started");
    }
```
and we should change this in other callbacks as well (only addBreadcrumb if `options.isEnableActivityLifecycleBreadcrumbs()`, so adding breadcrumbs shouldn't depend on `performanceEnabled`)